src/Native/System.Security.Cryptography.Native/openssl.c

```diff
@@ -992,6 +1013,8 @@ static
 void
 LockingCallback(int mode, int n, const char* file, int line)
 {
+    (void)file, (void)line; // deliberately unused parameters
```

Comment:
This would still be needed since they'd be unused in release. Also, standard assert doesn't give us a way to format messages.